### ISSUE TITLE
SARSEN Error

### ISSUE THREAD

**Issue by S0sh0rt** [bopen/sarsen#12]
I have tested on both CentOS 7 and Windows 10. We are unable to use SARSEN because of this error:
sarsen gtc S1A_IW_GRDH_1SDV_20210519T094926_20210519T094951_037955_047ACD_7336.zip "IW2/VV" GLO20_WGS84_clipped.tif
Traceback (most recent call last):
  File "/opt/anaconda3/envs/SARSEN/bin/sarsen", line 8, in <module>
    sys.exit(app())
  File "/opt/anaconda3/envs/SARSEN/lib/python3.10/site-packages/typer/main.py", line 214, in __call__
    return get_command(self)(*args, **kwargs)
  File "/opt/anaconda3/envs/SARSEN/lib/python3.10/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
  File "/opt/anaconda3/envs/SARSEN/lib/python3.10/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
  File "/opt/anaconda3/envs/SARSEN/lib/python3.10/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/opt/anaconda3/envs/SARSEN/lib/python3.10/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/opt/anaconda3/envs/SARSEN/lib/python3.10/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
  File "/opt/anaconda3/envs/SARSEN/lib/python3.10/site-packages/typer/main.py", line 500, in wrapper
    return callback(**use_params)  # type: ignore
  File "/opt/anaconda3/envs/SARSEN/lib/python3.10/site-packages/sarsen/__main__.py", line 17, in gtc
    apps.backward_geocode_sentinel1(
  File "/opt/anaconda3/envs/SARSEN/lib/python3.10/site-packages/sarsen/apps.py", line 84, in backward_geocode_sentinel1
    measurement_ds = xr.open_dataset(
  File "/opt/anaconda3/envs/SARSEN/lib/python3.10/site-packages/xarray/backends/api.py", line 495, in open_dataset
    backend_ds = backend.open_dataset(
  File "/opt/anaconda3/envs/SARSEN/lib/python3.10/site-packages/xarray_sentinel/xarray_backends.py", line 18, in open_dataset
    ds = sentinel1.open_sentinel1_dataset(
  File "/opt/anaconda3/envs/SARSEN/lib/python3.10/site-packages/xarray_sentinel/sentinel1.py", line 715, in open_sentinel1_dataset
    product_attrs, product_files = esa_safe.parse_manifest_sentinel1(file)
  File "/opt/anaconda3/envs/SARSEN/lib/python3.10/site-packages/xarray_sentinel/esa_safe.py", line 111, in parse_manifest_sentinel1
    manifest = ElementTree.parse(manifest_path)
  File "/opt/anaconda3/envs/SARSEN/lib/python3.10/xml/etree/ElementTree.py", line 1229, in parse
    tree.parse(source, parser)
  File "/opt/anaconda3/envs/SARSEN/lib/python3.10/xml/etree/ElementTree.py", line 580, in parse
    self._root = parser._parse_whole(source)
xml.etree.ElementTree.ParseError: not well-formed (invalid token): line 1, column 2

**Comment by alexamici** [comment]
@S0sh0rt the `not well-formed` error id from `ElementTree` and it hints that the manifest file is not a valid XML. It looks like a corrupt zip, or the file is really badly broken for other reasons.

Not a *Sarsen* issue. I'll close.

BTW: GRD images have the `IW/VV` group, not the `IW2/VV` one.

**Comment by S0sh0rt** [comment]
I corrected my comment above. I entered the wrong command that had the wrong parameter: IW2 that I had tested.  I use IW.

SARSEN works using the same file after unzipping. There may be an issue with my environment that is causing the zipped file to error.

FYI: SARSEN results are ~identical geometrically and similar radiometrically compared to S1TBX and GAMMA.  SARSEN is about 3x more CPU for GTC, and 10x more for RTC, than S1TBX.
